Epona/include/Epona/FloatingPoint.hpp

```diff
@@ -12,7 +12,7 @@ namespace epona
 {
 namespace fp
 {
-constexpr double g_floatingPointThreshold = 1e-6;
+constexpr double g_floatingPointThreshold = 1e-3;
```

Comment:
This is kind of unrelated. But it is necessary for the GJK and EPA stability, and I already have a task to make it configurable.